lib/ansible/modules/network/aci/aci_bd_dhcp_label.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_bd_dhcp_label
+short_description: Manage DHCP Label on Cisco ACI fabrics (dhcp:Lbl)
+description:
+- Manage DHCP Label on Cisco ACI fabrics (dhcp:Lbl)
+- More information from the internal APIC class
+  I(dhcp:Lbl) at U(https://pubhub-prod.s3.amazonaws.com/media/apic-mim-ref/docs/MO-dhcpLbl.html).
+author:
+- sig9 (sig9@sig9.org)
+version_added: '3.0'
+requirements:
+- ACI Fabric 1.0(3f)+
+notes:
+- A DHCP relay label contains a name for the label, the scope, and a DHCP option policy. The scope is the owner of the relay server and the DHCP option policy supplies DHCP clients with configuration parameters such as domain, nameserver, and subnet router addresses. 
+options:
+  bd:
+    description:
+    - The name of the Bridge Domain.
+  description:
+    description:
+    - Specifies a description of the policy definition.
+  dhcp_label:
+    description:
+    - The name of the DHCP Relay Label.
+  dhcp_option:
+    description:
+    - The DHCP option is used to supply DHCP clients with configuration parameters such as a domain, name server, subnet, and network address.
+  owner:
+    description:
+    - Represents the target relay servers ownership. 
+  tenant:
+    description:
+    - The name of the Tenant.
+    aliases: [ tenant_name ]
+'''
+
+EXAMPLES = r''' # '''
+
+RETURN = r''' # '''
+
+from ansible.module_utils.aci import ACIModule, aci_argument_spec
```

Comment:
line 54 should be...

from ansible.module_utils.network.aci.aci import ACIModule, aci_argument_spec

this appears to be why the sanity checks are failing

Reply:
Yes, that file has moved in the meantime. That's also why we have that conflict.